code/datums/martial/sleeping_carp.dm

```diff
@@ -342,7 +342,7 @@
 		oh_no_armor += human_carp_user.run_armor_check(potential_chest, MELEE)
 
 	// We also consider whether our footwear is appropriate
-	if(is_it_the_shoes.clothing_flags & CARP_STYLE_FACTOR)
+	if(is_it_the_shoes && is_it_the_shoes.clothing_flags & CARP_STYLE_FACTOR)
```

Comment:
This will runtime if you are not wearing clothing in your shoe slot

Reply:
👍